site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -145,7 +196,7 @@ const CreateConsequenceServices = (props: CreateConsequenceServicesProps): React
 
     useEffect(() => {
         const loadOptions = async () => {
-            const serviceDataToShow: RouteWithServiceInfo[] = [];
+            const serviceDataToShow: RouteWithServiceInfo[] | RouteWithServiceInfoPreformatted[] = [];
```

Comment:
```suggestion
            const serviceDataToShow: RouteWithServiceInfoPreformatted[] = [];
```

Can we update the type like so, then on line 226 and 236 we don't have to cast the type. I've tested it locally and looks like the map still works, not sure if there was a specific reason for including both types?

Reply:
done